### ISSUE TITLE
Unable to resolve module accessibilityinfo 

### ISSUE THREAD

**Issue by dhaval-apexon** [karolbarkowski/react-native-progress-bar-modest#1]
![Simulator Screen Shot - iPhone 11 - 2023-02-28 at 12 41 22](https://user-images.githubusercontent.com/114398356/221780260-b6c16908-6974-42d7-961d-10709b9b714d.png)
